databases/postgresql-supabase/README.md

```diff
@@ -88,10 +88,16 @@ Your database is now in sync with your schema.
 
 ### 3. Run the script
 
-To run the script `script.ts`, run the following command: 
+For the script to work, you first need to execute the [seed](./prisma/seed.ts) script to seed your database. You can do that using the following command:
+
+```
+npx prisma db seed
+```
```

Comment:
Is this always needed or does `migrate dev` maybe take care of that if you actually use it?

Reply:
AFAIK it should run with the migrate or generate script, but it did not. Hence I included it. 

![image](https://github.com/prisma/prisma-examples/assets/64993082/25e3c4ea-bb8f-4a20-9ffa-de1ecb605203)

Reply:
It says [here](https://www.prisma.io/docs/orm/prisma-migrate/workflows/seeding#integrated-seeding-with-prisma-migrate) it should run automatically given the `seed` property exists.

Reply:
Yeah, I recently had a conversation with @nikolasburk that the docs are not clear enough there. It is only run in `migrate dev` when that also runs a reset behind the scenes because of drift and you accepting that. It does _not_ generally run it.

So our instructions were definitely missing that until now.

Reply:
@janpio so does this look good to you? If so let me or @jharrell know so we can merge it.